README.rst

```diff
@@ -396,6 +411,25 @@ Usage example as a context-manager:
             signal=openwisp_signal,
         )
 
+``openwisp_utils.tests.TimeLoggingTestRunner``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. figure:: https://raw.githubusercontent.com/openwisp/openwisp-utils/master/docs/TimeLoggingTestRunner.png
+  :align: center
+
+This class extends the `default test runner provided by Django <https://docs.djangoproject.com/en/dev/ref/settings/#std:setting-TEST_RUNNER>`_
+and logs the time spent by each test, making it easier to spot slow tests by highlighting
+time taken by it in yellow (time shall be highlighted in red if it crosses the second threshold).
+
+By default tests are considered slow if they take more than 0.3 seconds but you can control
+this with `OPENWISP_SLOW_TEST_THRESHOLD <#openwisp_slow_test_threshold>`_
```

Comment:
You forgot a final dot here, I'm going to add it..

Reply:
Thanks :+1: